### PR TITLE
[#76059] Abbreviate weekday name to avoid wrapping

### PR DESCRIPTION
**Pull request by drench**
Before timeline nav before

After timeline nav after

### CODE REVIEW

File: app/views/facility_reservations/timeline.html.haml [hunk 1]
```diff
@@ -16,7 +16,7 @@
       = link_to "&laquo;".html_safe,
         timeline_facility_reservations_path(current_facility, date: format_usa_date(@display_date - 1.day)), id: "reservation_left"
       #reservation_date
-        = human_date_with_day(@display_date)
+        = l(@display_date.to_date, format: :timeline_navigation)
```
Comment (jhanggi) [inline app/views/facility_reservations/timeline.html.haml]: Just curious, would it be possible to pull the `#to_date` a little further up so `@display_date` is always already a date?

**Comment by jhanggi** [comment]
One potential refactoring, but otherwise looks good.

**Comment by drench** [comment]
Refactored. I also reformatted the public timeline partial to bring it closer to the timeline partial. We may be able to extract common pieces of these at some point, given their similarities.

**Comment by jhanggi** [comment]
:+1:

**Pull request closed without merging**